production/docker/config/loki-frontend-config.yaml

```diff
@@ -0,0 +1,42 @@
+auth_enabled: false
+
+http_prefix:
+
+server:
+  http_listen_port: 3100
+  log_level: debug
+
+limits_config:
+  max_cache_freshness_per_query: '10m'
+
+query_range:
+    # make queries more cache-able by aligning them with their step intervals
+    align_queries_with_step: true
+    max_retries: 5
+    # parallelize queries in 15min intervals
+    split_queries_by_interval: 15m
```

Comment:
I feel like we could also // by shard using `parallelise_shardable_queries: true`